### PR TITLE
Add some AWS database backup and restore docs

### PR DESCRIPTION
**Pull request by issyl0**
- In our upcoming New World of AWS, how we manage databases has changed. We have a lot of docs to write for everything, so we thought we'd make a start.

https://trello.com/c/vBkLAvM8/852-aws-backups-review-processes-and-add-documentation

### CODE REVIEW

**Comment by surminus** [comment]
Please ensure the docs have the correct headers, and the suffix is correct (`html.md`).

File: source/manual/aws-backup-and-restore-introduction.html.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+owner_slack: "#govuk-infrastructure"
+title: Backup and restore databases in RDS - introduction
```
Comment (surminus) [inline source/manual/aws-backup-and-restore-introduction.html.md]: Is this article specifically about RDS or AWS?

File: source/manual/aws-backup-and-restore-introduction.html.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+section: AWS
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2017-12-05
+review_in: 1 month
+---
+
+# Backup and restore databases in AWS - Introduction
+
+In AWS, we have five datastores: MySQL, PostgreSQL, ElasticSearch, MongoDB, and
```
Comment (surminus) [inline source/manual/aws-backup-and-restore-introduction.html.md]: Drop the "In AWS".

File: source/manual/aws-backup-and-restore-introduction.html.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Redis. Whether we back them up, restore them, and how we do so depends on how
```
Comment (surminus) [inline source/manual/aws-backup-and-restore-introduction.html.md]: This sentence isn't really needed. Details about how they're managed can go below in short summary sections.

File: source/manual/aws-backup-and-restore-introduction.html.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+they are managed in AWS.
+
+MySQL and PostgreSQL databases are fully managed through Amazon RDS.
```
Comment (surminus) [inline source/manual/aws-backup-and-restore-introduction.html.md]: Link to RDS docs.

File: source/manual/aws-howto-backup-and-restore-rds.html.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+section: AWS
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2017-12-05
+review_in: 1 month
+---
+
+## Backup and restore for RDS databases - howto
+
+Backups of RDS instances are taken nightly. They are stored in Amazon S3. SQL
```
Comment (surminus) [inline source/manual/aws-howto-backup-and-restore-rds.html.md]: Link to where this is configured in Terraform.

**Comment by surminus** [comment]
I think these docs look good. When 2ndline test the process they will probably change organically.

File: source/manual/aws-backup-and-restore-introduction.html.md (hunk 1 above)
Comment (issyl0) [reply source/manual/aws-backup-and-restore-introduction.html.md]: Oh. I meant AWS.

**Comment by tijmenb** [comment]
Would it make sense to merge the 2 pages? The intro is also relevant for the howto (we should probably skip the `- howto` at the end for consistency with https://docs.publishing.service.gov.uk/manual/docs-style-guide.html#title)

**Comment by issyl0** [comment]
@tijmenb Eventually the plan is to merge them, but for now while we're writing them and deciding how we're going to write all the docs (we need to write docs for ElasticSearch backups too, for example), I was going to keep them separate.

**Comment by issyl0** [comment]
@tijmenb For context, [this](https://github.com/alphagov/govuk-developer-docs/commit/f890330b6f7195aaaab2963271c0839a22bfaaf0) is our outline (and sort of steps for how to do the backups themselves) of the stuff we need to write about

**Comment by issyl0** [comment]
@surminus I fixed all the issues you pointed out - thanks.

**Comment by surminus** [comment]
Reading through this again, I think it may work better if we have a general document on "Datastores in AWS" and link to specific backups and restores from there, rather than having an introduction to backups and restores. I suspect people will search for the specific thing, and either end up on a page about how we run databases in AWS, or the specific restore guide.

**Comment by issyl0** [comment]
I'll close this for now and re-think.

**Comment by surminus** [comment]
I think how to restore stuff from RDS is absolutely fine, so there is nothing stopping that going in.

**Comment by issyl0** [comment]
Oh, OK. I'll remove the introduction stuff and re-think that separately then.

**Comment by issyl0** [comment]
Done! And I think I've adhered to the style guide by removing the "howto" from the page title.

**Pull request merged**